fml/message_loop_unittests.cc

```diff
@@ -75,7 +75,12 @@ TEST(MessageLoop, NonDelayedTasksAreRunInOrder) {
     auto& loop = fml::MessageLoop::GetCurrent();
     size_t current = 0;
     for (size_t i = 0; i < count; i++) {
-      loop.GetTaskRunner()->PostTask([&terminated, i, &current, count]() {
+      loop.GetTaskRunner()->PostTask([&terminated, i, &current
+#if OS_WIN
```

Comment:
I have no idea what issue this is trying to work around on Windows. This is not maintainable and I am sure the next time anyone tries to modify such TUs, he/she is going to miss one of these preprocessor guards.